KsApi/models/graphql/adapters/CommentsEnvelope+GraphCommentsEnvelopeTests.swift

```diff
@@ -1,10 +1,18 @@
+import ApolloAPI
 @testable import KsApi
 import Prelude
 import XCTest
 
 final class CommentsEnvelope_GraphCommentsEnvelopeTests: XCTestCase {
   func testProjectComments() {
-    let envelope = CommentsEnvelope.commentsEnvelope(from: .template)
+    // FIXME: Add templates for Apollo models and update these tests.
```

Comment:
Will solve this in a subsequent PR, I'm hoping there's a way that we can code-generate these templates.